src/interactions/slashCommands/mixins/CommandChannelOptionBase.ts

```diff
@@ -0,0 +1,44 @@
+import type { ChannelType } from 'discord-api-types/v9';
+import ow from 'ow';
+import type { ToAPIApplicationCommandOptions } from '../../..';
+import { SlashCommandOptionBase } from './CommandOptionBase';
+
+// Only allow valid channel types to be used. (This can't be dynamic because const enums are erased at runtime)
+const maxChannelType = 13;
+const channelTypePredicate = ow.number.greaterThanOrEqual(0).lessThanOrEqual(maxChannelType);
```

Comment:
These are not the types you can pass in... Isn't it just guild channels? Won't this include DM channels too?